py-polars/docs/requirements-docs.txt

```diff
@@ -4,7 +4,7 @@ numpy
 pandas
 pyarrow
 
-hypothesis==6.97.4
+hypothesis==6.92.1
```

Comment:
This is not supposed to happen, looks like you had an issue rebasing.

Reply:
Indeed, I've changed that back